client/src/app/api/rest.ts

```diff
@@ -846,3 +854,23 @@ export const updatePlatform = (platform: SourcePlatform) =>
 // success with code 204 and therefore no response content
 export const deletePlatform = (id: number) =>
   axios.delete<void>(`${PLATFORMS}/${id}`);
+// Manifests
+//
+
+export const getManifests = (): Promise<Manifest[]> =>
+  axios.get(MANIFESTS).then(({ data }) => data);
+
+export const getManifestById = (id: number | string): Promise<Manifest> =>
+  axios.get(`${MANIFESTS}/${id}`).then(({ data }) => data);
+
+// success with code 201 and created entity as response data
+export const createManifest = (manifest: New<Manifest>) =>
+  axios.post<Manifest>(MANIFESTS, manifest).then((res) => res.data);
+
+// success with code 204 and therefore no response content
+export const updateManifest = (manifest: Manifest): Promise<void> =>
+  axios.put(`${MANIFESTS}/${manifest.id}`, manifest);
+
+// success with code 204 and therefore no response content
+export const deleteManifest = (id: number): Promise<void> =>
+  axios.delete(`${MANIFESTS}/${id}`);
```

Comment:
Frontend will not need to deal with these at all.  we'll just access via the `/applications/:id/manifest` endpoint.